terraform/modules/aws/card-frontend.tf

```diff
@@ -113,6 +113,12 @@ resource "aws_wafv2_web_acl" "card_frontend" {
     allow {}
   }
 
+  lifecycle {
```

Comment:
Are these meant to be in this PR?

Reply:
Yeah, it reflects the current state - I was going to do them in a separate PR but I applied it all at once, I'm fine to remove it though